precompiles/avs/IAVSManager.sol

```diff
@@ -7,12 +7,26 @@ address constant AVSMANAGER_PRECOMPILE_ADDRESS = 0x00000000000000000000000000000
 IAVSManager constant AVSMANAGER_CONTRACT = IAVSManager(
```

Comment:
Please `forge fmt` this file with `exocore-contracts/foundry.toml` configuration.

Reply:
fixed